ntbs-service/wwwroot/source/Components/ValidateDate.ts

```diff
@@ -31,7 +31,7 @@ const ValidateDate = Vue.extend({
                 "RequestVerificationToken": (<HTMLInputElement>document.querySelector('[name="__RequestVerificationToken"]')).value
             }
 
-            axios.post(`/${this.$props.model}s/Edit/Validate${this.$props.model}Date?key=${this.$props.property}&day=${date.day}&month=${date.month}&year=${date.year}`, 
+            axios.post(`${this.$props.model}/Validate${this.$props.model}Date?key=${this.$props.property}&day=${date.day}&month=${date.month}&year=${date.year}`, 

```

Comment:
Thanks for making this sweet change here to move it to the notification-centric structure! Could you make sure to let the team know new pages should this once you've merged? :)